systemd/crc-cluster-status.sh

```diff
@@ -4,12 +4,33 @@ set -x
 
 export KUBECONFIG=/opt/kubeconfig
 
+if [ ! -f /opt/crc/pass_kubeadmin ]; then
+    echo "kubeadmin password file not found"
+    exit 1
+fi
+
+PASS_KUBEADMIN="$(cat /opt/crc/pass_kubeadmin)"
+
 rm -rf /tmp/.crc-cluster-ready
 
 if ! oc adm wait-for-stable-cluster --minimum-stable-period=1m --timeout=10m; then
     exit 1
 fi
 
+set +x
+echo "Logging into OpenShift with kubeadmin user to update $KUBECONFIG"
+COUNTER=1
+MAXIMUM_LOGIN_RETRY=10
+until oc login --insecure-skip-tls-verify=true -u kubeadmin -p "$PASS_KUBEADMIN" https://api.crc.testing:6443 > /dev/null 2>&1; do
```

Comment:
Not directly related to this change, but since this runs on the cluster, this most likely could use 
```
    --certificate-authority='':
	Path to a cert file for the certificate authority
```
instead of skipping TLS verification altogether.